tests/test_setup.py

```diff
@@ -0,0 +1,304 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+Usage:
+This file defines tasks for building Protos, webhost and extensions
+
+To use these tasks, you can run the following commands:
+
+1. Build protos:
+   invoke -c test_setup build_protos
+
+2. Set up the Azure Functions Web Host:
+   invoke -c test_setup webhost
+
+3. Install WebJobs extensions:
+   invoke -c test_setup extensions
+"""
+
+import glob
+import json
+import os
+import pathlib
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import urllib.request
+import zipfile
+from distutils import dir_util
+
+from invoke import task
+
+from tests.utils.constants import EXTENSIONS_CSPROJ_TEMPLATE, NUGET_CONFIG
+
+ROOT_DIR = pathlib.Path(__file__).parent.parent
+BUILD_DIR = ROOT_DIR / 'build'
+WEBHOST_GITHUB_API = "https://api.github.com/repos/Azure/azure-functions-host"
+WEBHOST_GIT_REPO = "https://github.com/Azure/azure-functions-host/archive"
+WEBHOST_TAG_PREFIX = "v4."

```

Comment:
These are added to constants.py, you can reference them from there?

Reply:
Since they are not used anywhere else I kept them in this file itself. If we need them in any other file, then lets move only that constant